conf.py

```diff
@@ -18,7 +18,7 @@
 # -- Project information -----------------------------------------------------
 
 project = 'Firely .NET SDK'
-copyright = '2021, Firely'
+copyright = '2023, Firely'
```

Comment:
I am actually never updating the date, since the copyright on this stuff is older than today - we don't want anyone to say that they have copyrighted this stuff 5 years ago, and not us?

Reply:
Ha! I am wrong:

https://opensource.stackexchange.com/questions/6389/how-do-the-years-specified-in-a-copyright-statement-work